iFixit/react-commerce
# Collections: Add meta title to strapi

- Adds a new field, `metaTitle` to Strapi. It has a 130 character limit based on the spec in this issue: https://github.com/iFixit/ifixit/issues/39157
- Uses the `metaTitle` field in the `<title>` of the `<head>`, and nowhere else on the page.

## QA
Pages with a value in `metaTitle` should have the value in the `<title>` tag, otherwise it should default to the normal device title. You can edit the `metaTitle` in Strapi here: https://collections--add-meta-title-to-strapi.govinor.com/admin

CC @ifixitmaxb

Closes #539
